File: src/main/Java/boj_step15/boj13241.java
import java.io.*;
import java.util.StringTokenizer;

public class boj13241 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        long a = Integer.parseInt(tokenizer.nextToken());
        long b = Integer.parseInt(tokenizer.nextToken());
        writer.append(String.valueOf((a * b) / gcd(a, b))).close();
    }
    public static long gcd (long a, long b) { if (b != 0) return gcd (b, a % b); else return a; }
}

File: src/main/Kotlin/boj_step15/boj13241.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (a, b) = r.readLine().split(" ").map { it.toLong() }
    w.write(((a * b) / gcd(a, b)).toString())
    w.close()
}

private fun gcd(a: Long, b: Long): Long = if (b != 0L) gcd(b, a % b) else a
